lexico/utils.py

```diff
@@ -279,3 +286,17 @@ def tabulate_words(formatted_data):
     '''Tabulates the given words for user viewing.'''
     headers = ['Word', 'Lookups', 'Created At', 'Last Lookup']
     return tabulate(formatted_data, headers=headers)
+
+###############################################################################
+################################## TRANSLATE ##################################
+###############################################################################
+
+def fetch_translations(word_to_translate):
+    languages_to_use = ["de","es","fr","hi","ru"]
+    translator = Translator()
+    word_translations = []
+    for language in languages_to_use:
+        word_in_this_language = translator.translate(word_to_translate, dest=language)
+        word_with_language_info = word_in_this_language.text + " (" + language + ")"
+        word_translations.append(word_with_language_info)
+    return word_translations
```

Comment:
Always remember to put a newline(or "Press Enter") at the end of source code files.

Reply:
Thanks for the advice. I'll keep that in mind in the future.